packages/ui/Slottable/tests/useSlots-specs.js

```diff
@@ -9,14 +9,15 @@ import useSlots from '../useSlots';
 describe('useSlots', () => {
 
 	test('should distribute children with a \'slot\' property', () => {
-		function Component (props) {
-			const {a, b, c} = useSlots({slots: ['a', 'b', 'c'], props});
+		// eslint-disable-next-line enact/prop-types
```

Comment:
Can you add a test that `children` is actually modified on output?

Reply:
Done!